### PR TITLE
Ensemble learning techniques to improve Machine Learning

### PR DESCRIPTION
**Pull request by collins-a**
Article on ensemble learning as index.md.
Hero image as hero.jpg.
Other images include bagging.png, stacking.png and stackingregression.png

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by collins-a** [comment]
This fixes #728

File: articles/ensemble-learning/index.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+3. What follows is the creation of base models on each of the bootstrapped subsets. It is worth noting that the models run independently and in parallel to each other.
+
+4. The final step involves determining the final predictions by combining the results from all the models.
+
+The image below will aid in the understanding of the bagging process.
+
+![bagging](/engineering-education/ensemble-learning/bagging.png)
+
+Bagging process
```
Comment (lalith1403) [inline articles/ensemble-learning/index.md:55]: _Bagging process_

**Review by lalith1403 — APPROVED** [review]
I have made a minor change. Note that. Very well written article on ensemble learning. Can you write an article on how ensemble learning affects bias and variance? It would be a great topic for the next article.

**Comment by collins-a** [comment]
@lalith1403 thank you for the review. The changes have been noted. I'll implement the italics on the article I've uploaded just now. And I'll definitely have ensemble x bias x variance as my next topic suggestion

**Comment by hectorkambow** [comment]
Plagiarism check done

**Pull request merged**